exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -36,43 +36,42 @@ class Exporter # rubocop:disable Metrics/ClassLength
         private_constant(:ERROR_MESSAGE_INVALID_HEADERS)
 
         def self.ssl_verify_mode
-          if ENV.key?('OTEL_RUBY_EXPORTER_OTLP_SSL_VERIFY_PEER')
-            OpenSSL::SSL::VERIFY_PEER
-          elsif ENV.key?('OTEL_RUBY_EXPORTER_OTLP_SSL_VERIFY_NONE')
-            OpenSSL::SSL::VERIFY_NONE
-          else
-            OpenSSL::SSL::VERIFY_PEER
-          end
+          return OpenSSL::SSL::VERIFY_PEER if ENV.key?('OTEL_RUBY_EXPORTER_OTLP_SSL_VERIFY_PEER')
+          return OpenSSL::SSL::VERIFY_NONE if ENV.key?('OTEL_RUBY_EXPORTER_OTLP_SSL_VERIFY_NONE')
+
+          OpenSSL::SSL::VERIFY_PEER
```

Comment:
It looks cleaner, easier to read. Might be subjective :)
regardless of that, I opened a discussion about this method. https://github.com/open-telemetry/opentelemetry-ruby/discussions/1350

Reply:
I like this style when it’s short, but this hits around 80 characters. When it gets this long, I find it’s too much to hold in my head before I know if I can proceed to the next line or if I must jump back up the call stack. The previous version of this code let me know if I could progress through the file more quickly as I read, and thus feels nicer (for the same reasons people like to feel as though they are “making progress” on a trip via the scenic route, even if sitting in slow traffic is technically faster because it’s more direct).